src/sass/_variables.scss

```diff
@@ -6,6 +6,9 @@ $footer-bg: #363636;
 
 $primary: #d58e32;
 
+$white: #ffffff;
```

Comment:
Bad name for a color variable. What if the client will request to change the color of the background to red - some anomaly will occur like the following one: `$white: #ff0000;`. Better names will be such as `category-menu-bg-color` or more globally `main-bg-color`.